.github/workflows/tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
```

Comment:
Not that it's critical for tests, but since Lambda only supports 10.x and 12.x why make this change? https://docs.aws.amazon.com/lambda/latest/dg/lambda-nodejs.html

Reply:
For middy v2 I'm using esm and optional chaining. These are only enabled by default in node v14. middy v2 will not be released until AWS supports node v14, likely in Apr/May.

Reply:
Why not use Babel to transpile to a Node 10 compatible version? https://github.com/middyjs/middy/blob/master/packages/core/.babelrc#L5

Looks like v1 core is still going all the way back to Node 6 which is probably worth updating regardless

Reply:
I try to avoid transpiler due to their configuration complexity and the increase maintenance they require. I have been reflecting more on this. During publish I will likely transpile a commonjs flavour to node v12 to allow for an easier upgrade to v2. I'll need to do more research and testing to best do this without complicating our testing suite.

AWS will likely deprecate node v10 in may this year.